### PR TITLE
Add support for local metric collection/viewing

### PR DESCRIPTION
**Pull request by jackcmay**
#### Summary of Changes

Fixes #

### CODE REVIEW

**Comment by mvines** [comment]
Seeing this patch, the goodness under `scripts/local-metrics/` seems to feel like it would fit better at `metrics/scripts`. wdyt?

**Comment by mvines** [comment]
@danpaul000 should review this too, I added him

File: metrics/scripts/enable.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+export INFLUX_HOST=http://localhost:8086
+export INFLUX_DATABASE=local
+export INFLUX_USERNAME=admin
+export INFLUX_PASSWORD=admin
+
+export SOLANA_METRICS_CONFIG="host=$INFLUX_HOST,db=$INFLUX_DATABASE,u=$INFLUX_USERNAME,p=$INFLUX_PASSWORD"
```
Comment (mvines) [inline metrics/scripts/enable.sh]: how about just setting this env var, then running `scripts/configure-metrics.sh` to set all the INFLUX vars...less places to change in the future

File: metrics/scripts/start.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+docker run \
+  --detach \
+  --name=influxdb \
+  --net=influxdb \
+  --publish 8086:8086 \
+  --user "$(id -u):$(id -g)" \
+  --volume "$PWD"/influxdb.conf:/etc/influxdb/influxdb.conf:ro \
+  --volume "$PWD"/lib/influxdb:/var/lib/influxdb \
+  -e INFLUXDB_DB=local \
+	-e INFLUXDB_ADMIN_USER=admin \
```
Comment (mvines) [inline metrics/scripts/start.sh]: nit: fix indent.  Also use long arg, `--env` for consistency

File: metrics/scripts/enable.sh (hunk 1 above)
Comment (jackcmay) [reply metrics/scripts/enable.sh]: Seemed weird to call back down to configure-metrics.sh.  Maybe should move `configure-metrics.sh` and `metrics-write-datapoint.sh` to /metrics/scripts

Comment (mvines) [reply metrics/scripts/enable.sh]: Moving seems better longer term but that's gonna cause some painful churn.  I vote to call down for now with a TODO in this file to consider moving those two scripts later

**Comment by jackcmay** [comment]
@mvines Think we should move this local metrics stuff into `./metrics/scripts/local`?

**Comment by mvines** [comment]
> @mvines Think we should move this local metrics stuff into `./metrics/scripts/local`?

Too nested!  Instead I want to generalize `/metrics/scripts` into something that works for metrics.solana.com and testnet-specific metrics instances as well (probably defaulting to "local")

**Pull request merged**